libs/src/svkImageReader2.h

```diff
@@ -65,20 +64,46 @@ class svkImageReader2 : public vtkImageReader2
 
         vtkTypeRevisionMacro( svkImageReader2, vtkImageReader2);
 
+        typedef enum {
+            UNDEFINED = -1,
+            DICOM_MRS=0,
+            DICOM_MRI,
+            DICOM_ENHANCED_MRI,
+            DICOM_SEGMENTATION,
+            DICOM_RAW,
+            SIEMENS_RDA,
+            VARIAN_FID,
+            VARIAN_FDF,
+            GE_PFILE,
+            GE_SDBM,
+            GE_POSTAGE_STAMP,
+            GE_SIGNA_LX2,
+            GE_SIGNA_5X,
+            LC_MODEL_COORD, 
+            LC_MODEL_CSV, 
+            BRUKER_MRS,
+            DDF,
+            IDF,
+            LAST_TYPE = IDF
+        } ReaderType;
+
```

Comment:
What is the advantage of using enums over using the class name with GetClassName?